docs/_milestone_.md

```diff
@@ -0,0 +1,85 @@
+# December Milestone
+
+## Overview
+
+- Complete the epics:
+  - `Add Support for Partitioned Database`
+  - `Inclusion of related models`
+  - `Production deployment`
+
```

Comment:
I'd also like use to complete spike on migrating auth & auth from LB3 to LB4, so that we can start planning work on the migration guide and/or tooling for 2020Q1